cpu/mips32r2_common/Makefile.include

```diff
@@ -4,5 +4,16 @@ export APP_START=0x80000000
 
 include $(RIOTMAKE)/arch/mips.inc.mk
 
-export LINKFLAGS += -Tuhi32.ld
 export USEMODULE += periph
+export USEMODULE += newlib
+
+ifeq ($(USE_UHI_SYSCALLS),1)
+  #Use UHI to handle syscall (default)
+  export LINKFLAGS += -luhi -Tuhi32.ld
+  export USEMODULE += newlib_syscalls_mips_uhi
+else
+  #Use RIOT to handle syscalls (VFS)
+  export LINKFLAGS += -Tuhi32.ld
```

Comment:
Yes, its the only one we have for a default mips32 implementation, it is copied from the toolchains in-built linker scripts and follows its naming convention.